src/nanosvg.h

```diff
@@ -29,6 +29,33 @@
 #ifndef NANOSVG_H
 #define NANOSVG_H
 
+#include <math.h>
```

Comment:
I called the naosvg library in the cpp file ,and reported some compile errors in VC 6.0, so add this statement